### ISSUE TITLE
Exported Segmentation (Segmentation mask 1.1) partially doesn't match segmentation in CVAT --> unuseable segmentations

### ISSUE THREAD

**Issue by LisaRentschler** [openvinotoolkit/cvat#4612]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)    
- [x] Updated CVAT to: Server version: 2.1, Core version: 5.0.1, Canvas version: 2.13.2, UI version: 1.37.0

### Expected Behaviour

Exported masks should look the same, as in CVAT after completing and merging.
Frame 259 in CVAT      
![20171031_259_CVAT](https://user-images.githubusercontent.com/99329946/166417806-e88b064c-ac8d-4988-9928-e8390dcf4f8a.PNG)       
Frame 259 after exporting     
![20171031_259_exported](https://user-images.githubusercontent.com/99329946/166417814-48af4d6f-d39e-4463-9cb2-faadf17f62a4.PNG)    

### Current Behaviour

Some segmentations are just wrong, although the frames are correctly annotated in CVAT and other frames within the same Job and Task are correct after exporting (see above, frame 259).   
Frame 258 in CVAT (after completing and merging) is correct        
![20171031_258_CVAT](https://user-images.githubusercontent.com/99329946/166418338-c4dca110-dcdf-4f70-9c65-0afda7057c67.PNG)    
Frame 258 after exporting is not correct     
![20171031_258_exported](https://user-images.githubusercontent.com/99329946/166418406-70a6473c-5c26-47a0-a311-da5cb1fde8b3.PNG)
     

### Steps to Reproduce (for bugs)

1.
1.
1.
1.

### Your Environment

- Git hash commit (`git log -1`): 119a27bbd6e589d140d49bc5c3f9098aee2dfff9
- Docker version `docker version` (e.g. Docker 17.0.05): 20.10.11, build dea9396
- Are you using Docker Swarm or Kubernetes? - No    
- Operating System and version (e.g. Linux, Windows, MacOS): Ubuntu 20.04.
- Code example or link to GitHub repo or gist to reproduce problem: n.a. 
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

**Comment by LisaRentschler** [comment]
Hi, thank you for adressing the problem. 
I wouldn't say the annotations are missing, I'd say they are just incorrect, because it's not like the base is pink and any other polygon is on top of it. For example: if I pull on the edge of the blue annotation in the lower left corner in the correct annotated CVAT image, it won't show pink underneath but the image itself without any annotation. Same with the orange polygon. There should be only one polygon, and it should be orange, not pink. The only coloured polygon which is layered upon another polygon ist the blue one in the middle. But this layer hierarchy is also correct in CVAT before i export it. Meaning the yellow polygonis in the background and the blue one is on top of it. 
As for the later two questions: Both isn't desirable (especially the different colours), but for our project it's not as crucial, as the overall analogousness of the correct annotated polygons in CVAT and in the exported set. This has to be the same before we can focus on colour problems. Maybe this might be another issue.

**Comment by BenediktRombach** [comment]
@zhiltsov-max Did this answer your questions, or are there any other informations we can provide to identify the problem? Even a workaround would help us, as we currently have a rather lage dataset annotated which we can not really use for training due to the export errors...

(for transparency, @LisaRentschler and I work for the same organization)
